src/test/java/org/openrewrite/java/testing/assertj/JUnitAssertThrowsToAssertExceptionTypeTest.java

```diff
@@ -63,4 +63,36 @@ public void throwsExceptionWithSpecificType() {
           )
         );
     }
+
+    @Test
+    void assertThrowsAssignment() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import static org.junit.jupiter.api.Assertions.assertThrows;
+              
+              public class SimpleExpectedExceptionTest {
+                  public void throwsExceptionWithSpecificType() {
+                      NullPointerException npe = assertThrows(NullPointerException.class, () -> {
+                          throw new NullPointerException();
+                      });
+                  }
+              }
+              """,
+            """
+              import static org.junit.jupiter.api.Assertions.assertThrows;
+              
+              public class SimpleExpectedExceptionTest {
+                  public void throwsExceptionWithSpecificType() {
+                      NullPointerException npe = assertThrows(NullPointerException.class, () -> {
+                          throw new NullPointerException();
+                      });
+                  }
+              }
+
```

Comment:
It looks like you want to assert no change is made; in such a case you only have to pass the first argument.
```suggestion
```